app/components/service/serviceController.js

```diff
@@ -335,6 +366,15 @@ function ($q, $scope, $transition$, $state, $location, $timeout, $anchorScroll,
 
   function initView() {
     var apiVersion = $scope.applicationState.endpoint.apiVersion;
+
+    PluginService.loggingPlugins(apiVersion < 1.25)
+    .then(function success(data){
+        $scope.availableLoggingDrivers = data;
+    })
+    .catch(function error(err) {
+      Notifications.error('Failure', err, 'Unable to retrieve logging drivers');
+    });        
+
     ServiceService.service($transition$.params().id)
```

Comment:
The `ServiceService.service($transition$.params().id)` and `PluginService.loggingPlugins(apiVersion < 1.25)` async queries should be grouped inside a `q.all()` directive.